blocks/hero/hero.css

```diff
@@ -0,0 +1,76 @@
+/* block specific CSS for hero banner goes here */
+
+main .hero-container>div {
+    max-width: unset;
+  }
+  
+  main .only-picture{
+    display: none;
+  }
+  
+  main .hero-container {
+    padding: 0;
+  }
+  
+  main .hero {
+    position: relative;
+    max-width: 327px;
+    margin: 0 auto;
+  }
+  
+  main .hero h1 {
+    color: white;
+    font-size: 22px;
```

Comment:
Can you add this font-size as a variable in style.css and use that variable here